docs/gwcs/fits_analog.rst

```diff
@@ -0,0 +1,101 @@
+.. _fits_equivalent_example
+
+FITS Equivalent WCS Example
+===========================
+
+The following example shows how to construct a GWCS object equivalent to
+a FITS imaging WCS without distortion, defined in this FITS imaging header::
+
+  WCSAXES =                    2 / Number of coordinate axes
+  WCSNAME = '47 Tuc     '        / Coordinate system title
+  CRPIX1  =               2048.0 / Pixel coordinate of reference point
+  CRPIX2  =               1024.0 / Pixel coordinate of reference point
+  PC1_1   =   1.290551569736E-05 / Coordinate transformation matrix element
+  PC1_2   =  5.9525007864732E-06 / Coordinate transformation matrix element
+  PC2_1   =  5.0226382102765E-06 / Coordinate transformation matrix element
+  PC2_2   = -1.2644844123757E-05 / Coordinate transformation matrix element
+  CDELT1  =                  1.0 / [deg] Coordinate increment at reference point
+  CDELT2  =                  1.0 / [deg] Coordinate increment at reference point
+  CUNIT1  = 'deg'                / Units of coordinate increment and value
+  CUNIT2  = 'deg'                / Units of coordinate increment and value
+  CTYPE1  = 'RA---TAN'           / TAN (gnomonic) projection + SIP distortions
+  CTYPE2  = 'DEC--TAN'           / TAN (gnomonic) projection + SIP distortions
+  CRVAL1  =        5.63056810618 / [deg] Coordinate value at reference point
+  CRVAL2  =      -72.05457184279 / [deg] Coordinate value at reference point
+  LONPOLE =                180.0 / [deg] Native longitude of celestial pole
+  LATPOLE =      -72.05457184279 / [deg] Native latitude of celestial pole
+  RADESYS = 'ICRS'                / Equatorial coordinate system
+
+
+The following imports are generally useful:
```

Comment:
I would suggest replacing this with something like
```suggestion
For the next example we need the following imports.
```

Reply:
I've done something similar to what you suggest.